docs/developer-guide/syntax-reference.rst

```diff
@@ -335,6 +335,50 @@ KSQL statements
       topics, and columns.
     - Don't use quotes around stream names or table names when you CREATE them.
 
+
+Quoted identifiers for source and column names
+----------------------------------------------
+
+Starting in KSQL 5.4, quoted identifiers in column names and source names are
+supported. If you have names that ksqlDB can't parse, enclose them in backtick
```

Comment:
Can we add a mention of case-sensitivity here? Quoted identifiers are more than just a workaround for source/column names that otherwise wouldn't parse as they also let users have full control over their source/column names. We get a fair number of questions about how to prevent KSQL from automatically converting column names to all-caps.

Reply:
@vcrfxia I finally got back to this, do these edits look okay?